src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card__wrapper {
+  margin: 0 16px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url('/src/images/imac.jpeg');
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $info-product-color;
+  }
+
+  &__code {
+    color: $identificator-color;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: $info-product-color;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      color: $identificator-color;
+    }
+
+    &-price {
+      font-size: 16px;
+      font-weight: 700;
+      color: $info-product-color;
+    }
+  }
+
+  &__button {
+    background-color: #00acdc;
+    font-size: 14px;
+    height: 40px;
+    width: 166px;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+
+    &:hover {
+      transition: 0.12s;
```

Comment:
the transition of what?) you should directly identify the property that animated